runtime/zeitgeist/src/lib.rs

```diff
@@ -174,10 +174,10 @@ impl Contains<RuntimeCall> for IsCallable {
                     create_market { scoring_rule: RikiddoSigmoidFeeMarketEma, .. } => false,
                     edit_market { scoring_rule: RikiddoSigmoidFeeMarketEma, .. } => false,
                     // Disable Court & SimpleDisputes dispute resolution mechanism
-                    create_market { dispute_mechanism: Court | SimpleDisputes, .. } => false,
-                    edit_market { dispute_mechanism: Court | SimpleDisputes, .. } => false,
+                    create_market { dispute_mechanism: Some(Court | SimpleDisputes), .. } => false,
+                    edit_market { dispute_mechanism: Some(Court | SimpleDisputes), .. } => false,
```

Comment:
Are you 100% sure that the `|` represents a logical or instead of a binary or in this case? (as enum elements can be represented as numbers).

Reply:
> Are you 100% sure that the `|` represents a logical or instead of a binary or in this case?

Yeah. This is the content of the following RFC: https://rust-lang.github.io/rfcs/2535-or-patterns.html.

> (as enum elements can be represented as numbers)

Not sure what you mean. Sure, variants can be represented as numbers, but I don't think Rust ever does that. I mean, what would `Court | SimpleDisputes` even be? What if the resulting number doesn't represent a variant of the enum?